lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -255,6 +256,7 @@ def add_extra_data(post, payment, options)
         post[:shopperStatement] = options[:shopper_statement] if options[:shopper_statement]
         post[:store] = options[:store] if options[:store]
 
+        add_shopper_data(post, payment, options)
```

Comment:
Was it under `add_extra_data` or `add_additional_data` before?

Reply:
Under `add_extra_data` https://github.com/activemerchant/active_merchant/blob/0013c6f29e5ad37b298643ff1ae82d2a56a70d60/lib/active_merchant/billing/gateways/adyen.rb#L231